modules/cnf-assigning-a-secondary-network-to-a-vrf.adoc

```diff
@@ -107,12 +94,48 @@ additional-network-1       14m
 There might be a delay before the CNO creates the CR.
 ====
 
-.Verifying that the additional VRF network attachment is successful
+.Verification
+
+To verify that the additional network is deployed and associated with the VRF, complete the following steps:
 
-To verify that the VRF CNI is correctly configured and the additional network attachment is attached, do the following:
+. Create a pod and assign it to the additional network with the VRF instance:
+
+.. Create a YAML file that defines the `Pod` resource:
++
+.Example `pod-additional-net.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+ name: pod-additional-net
+ annotations:
+   k8s.v1.cni.cncf.io/networks: '[
+       {
+               "name": "test-network-1" <1>
+       }
+ ]'
+spec:
+ containers:
+ - name: example-pod-1
+   command: ["/bin/bash", "-c", "sleep 9000000"]
+   image: centos:8
+----
+<1> Specify the name of the additional network with the VRF instance.
+
+.. Create the `Pod` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f pod-additional-net.yaml
+----
++
+.Example output
+[source,terminal]
+----
+pod/test-pod created
+----
 
-. Create a network that uses the VRF CNI.
-. Assign the network to a pod.
 . Verify that the pod network attachment is connected to the VRF additional network. Remote shell into the pod and run the following command:
```

Comment:
Is there a way to rephrase this to avoid using "remote shell" as a verbal phrase?